sbids/metrics/c2st.py

```diff
@@ -0,0 +1,128 @@
+
+from tqdm import tqdm
+import numpy as np
+import jax
+import jax.numpy as jnp
+import haiku as hk
+import optax
+from tensorflow_probability.substrates import jax as tfp
+tfd = tfp.distributions
+tfb = tfp.bijectors
+
+class Classifier_C2ST(hk.Module):
+
+  def __call__(self, x):
+
+    net_x = hk.Linear(8)(x) 
+    net_x = jax.nn.leaky_relu(net_x)
+    net_x = hk.Linear(16)(net_x) 
+    net_x = jax.nn.leaky_relu(net_x) 
+    net_x = hk.Linear(32)(net_x) 
+    net_x = jax.nn.leaky_relu(net_x) 
+    net_x = hk.Linear(16)(net_x) 
+    net_x = jax.nn.leaky_relu(net_x) 
+    net_x = hk.Linear(8)(net_x) 
+    net_x = jax.nn.leaky_relu(net_x) 
+    net_x = hk.Linear(1)(net_x) 
+    net_x = jax.nn.sigmoid(net_x).squeeze()
+    net_x = jnp.clip(net_x, 1e-5, 1.-1e-5)
+
+    return net_x
+
+
+def c2st(sample1,sample2,num_seed):
```

Comment:
same comment as before ^^, never pass a num_seed as argument to a jax function, but an actual prngkey